src/Swashbuckle.AspNetCore.SwaggerGen/XmlComments/XmlCommentsOperationFilter.cs

```diff
@@ -32,26 +37,28 @@ public void Apply(OpenApiOperation operation, OperationFilterContext context)
         private void ApplyControllerTags(OpenApiOperation operation, Type controllerType)
         {
             var typeMemberName = XmlCommentsNodeNameHelper.GetMemberNameForType(controllerType);
-            var responseNodes = _xmlNavigator.Select($"/doc/members/member[@name='{typeMemberName}']/response");
+
+            if (!_xmlDocMembers.TryGetValue(typeMemberName, out var methodNode)) return;
+
+            var responseNodes = methodNode.SelectChildren("response", "");
```

Comment:
Could add an extension method for `SelectChildren(string)` that calls `SelectChildren(value, "")` (via the constant I mentioned) to avoid the repeated need to pass through an empty namespace in the first place?